### PR TITLE
feature/796 Implement new health check

### PR DESCRIPTION
**Pull request by lewisdaly**
Part of: mojaloop/project#796

What we did:

- Utilized new HealthCheck implemented in central-services-shared and implemented a basic health check server, which checks the connectivity of kafka and mysql
- updated the swagger definition
- Added new unit and integration tests

### CODE REVIEW

**Comment by lewisdaly** [comment]
Integration tests are failing on CircleCI. They are passing when I run on my machine (I promise!), so I'll have to fix those before we continue.

File: src/api/root/handler.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ * @function getHealth
+ *
+ * @description Get the health of the service
+ *
+ * @param {*} request - the Hapi request object
+ * @param {*} h - the Hapi handler object
+ */
+const getHealth = async (request, h) => {
+  let responseBody
+  let responseCode = 200
```
Comment (rmothilal) [inline src/api/root/handler.js]: I think it’s best to move all response codes to the library

**Comment by mdebarros** [comment]
Note that the Integration tests failed on this. :(

File: src/handlers/lib/kafka/consumer.js [hunk 2]
```diff
@@ -120,15 +113,69 @@ const getConsumer = (topicName) => {
...
+
+      return resolve(metadata)
+    }
+
+    consumer.getMetadata({ topic, timeout: 3000 }, cb)
+  })
+}
+
+/**
+ * @function isConsumerConnected
```
Comment (mdebarros) [inline src/handlers/lib/kafka/consumer.js]: Same as previous PRs. I feel like this should just be `isConnected`.

**Comment by lewisdaly** [comment]
@mdebarros @rmothilal This one is (finally) ready to go again!

**Comment by lewisdaly** [comment]
@mdebarros done. Please merge when you are happy

**Pull request merged**